sync_cleared_for_campus.R

```diff
@@ -0,0 +1,170 @@
+library(tidyverse)
+library(dotenv)
+library(REDCapR)
+library(lubridate)
+library(rlang)
+
+source("functions.R")
+
+# uncomment to read data from production
+#load_dot_env('prod.env')
+
+# get_case_records suitability is questionable
+case_records <- redcap_read_oneshot(redcap_uri = Sys.getenv("URI"),
+            token = Sys.getenv("CASE_TOKEN"),
+            guess_max = 3000)$data %>%
+    mutate(case_emp_ufid = as.character(case_emp_ufid)) # type change for dplyr
```

Comment:
```suggestion
case_records <- redcap_read_oneshot(redcap_uri = Sys.getenv("URI"),
            token = Sys.getenv("CASE_TOKEN"),
            guess_max = 3000)$data %>%
    mutate(case_emp_ufid = as.character(case_emp_ufid)) %>% # type change for dplyr
    filter(redcap_event_name == "case_investigation_arm_1" & is.na(redcap_repeat_instrument) & !is.na(case_emp_ufid)) %>%
    select(record_id, redcap_event_name, case_emp_ufid, contains("cleared_for_campus"))
```

You are reading way more records (and columns) than you need here. I got rid of the records to reduce the noise. Not that this made the algorithm work. :-)